core/shared/src/main/scala/zio/ZSchedule.scala

```diff
@@ -114,43 +127,44 @@ trait ZSchedule[-R, -A, +B] extends Serializable { self =>
 
   /**
    * Returns the composition of this schedule and the specified schedule,
-   * by piping the output of this one into the input of the other, and summing
-   * delays produced by both.
+   * by piping the output of this one into the input of the other. Delays produced by this
+   * schedule will always run before the second schedule is evaluated.
```

Comment:
Maybe more generally, `Effects described by this schedule will always be executed before the effects described by the second schedule.`

Reply:
Much clearer 👍